services/blockchain-indexer/shared/dataService/network.js

```diff
@@ -33,6 +33,90 @@ const getNetworkStatus = async () => {
 	};
 };
 
+const getNetworkPeers = async params => {
```

Comment:
Can we please move the business logic to the business layer?

Reply:
Done!